lib/bindings/MQTTBinding.js

```diff
@@ -159,14 +159,15 @@ function unsubscribeAll(callback) {
  * @param {String} apiKey           API Key corresponding to the Devices configuration.
  * @param {String} deviceId         Id of the device to be updated.
  * @param {Object} device           Device object containing all the information about a device.
- * @param {Object} messageObj       JSON object sent using MQTT.
+ * @param {String} message          UL payload.
  */
-function multipleMeasures(apiKey, deviceId, device, messageObj) {
-    var updates = [];
+function multipleMeasures(apiKey, deviceId, device, message) {
```

Comment:
Is really needed to pass the `deviceId` to the `multipleMeasures()` function? (since it should be included in the `device` object passed)?